examples/times.txt

```diff
@@ -1,2 +1,3 @@
 60d306917dc051a2f9b1aa242d260d5940a819de sigints working (perhaps too complicated) all different: 0.938528 all identical: 0.411068 change on character 0.290748 100 iterations
 a33c3de0f91e0c68819af4c301dbf7b7cee5d2cf fixes for python3 based on chat example all different: 0.781105 all identical: 0.343822 change on character 0.178675 100 iterations
+a459fc8ccf93c248e2e6465517ea559fa2f1d629 Worked on the tests all different: 0.494262 all identical: 0.289213 change on character 0.144840 100 iterations
```

Comment:
Could you remove these changes, I usually just add these when we make a change that I expect to impact performance